geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/tables/AttributeTable.scala

```diff
@@ -76,7 +76,7 @@ object AttributeTable extends GeoMesaTable with Logging {
                     rowIdPrefix: String): FeatureWriterFn = {
 
     val indexesOfIndexedAttributes = indexedAttributes.map { a => sft.indexOf(a.getName) }
-    val attributesToIdx = indexesOfIndexedAttributes.zip(indexedAttributes)
+    val attributesToIdx = indexedAttributes.zip(indexesOfIndexedAttributes)
```

Comment:
yeah, i flipped it b/c I realized that there are a lot of places where I had to zipWithIndex and then reverse the result :P